desc/compute/_curve.py

```diff
@@ -501,6 +502,241 @@ def _x_sss_FourierXYZCurve(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="x",
+    label="\\mathbf{r}",
+    units="m",
+    units_long="meters",
+    description="Position vector along curve",
+    dim=3,
+    params=["X", "Y", "Z", "_knots", "_period", "_method"],
+    transforms={
+        "rotmat": [],
+        "shift": [],
+    },
+    profiles=[],
+    coordinates="s",
+    data=["s"],
+    parameterization="desc.geometry.curve.SplineXYZCurve",
+    basis="basis",
+)
+def _x_SplineXYZCurve(params, transforms, profiles, data, **kwargs):
+    xq = data["s"]
+
+    Xq = interp1d(
+        xq,
+        params["_knots"],
```

Comment:
these are the attributes of the XYZCurve, they are underscored I assumed because it is how it is accessed. Can I make them not underscored and it will just use the getter?

Reply:
yeah, if there is a property with the non-underscored name we should use that.